homeassistant/components/fastdotcom/__init__.py

```diff
@@ -40,38 +35,49 @@
 )
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+async def async_setup_platform(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
We need to move the import here as the user will never come to the platform.

Reply:
Your intital feedback was:

> Not yet as people can have a yaml config so it needs to be kept but can simply return true until the yaml config can be removed completely.

Should the create_task be moved here, as well as a redirect to the `async_step_import`?

Reply:
According to the documentation the integration loads from init so that's were we should start the import so all of it should be moved there.